src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -5499,6 +5499,221 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/sites/{siteId}/group-members':
+    get:
+      tags:
+        - sites
+      summary: List site group memberships
+      description: Gets a list of site group memberships for site **siteId**.
```

Comment:
...list of group membership...
Instead of "site group". Could be confused with groups of sites.